commonjs/angular-numbro.js

```diff
@@ -1,16 +1,16 @@
-window.require.define({"angular-numeraljs": function(exports, require, module) {
+window.require.define({"angular-numbro": function(exports, require, module) {
```

Comment:
'module' is defined but never used.<br>'require' is defined but never used.<br>'exports' is defined but never used.